app/views/admin/gdpr_delete_requests/index.html.erb

```diff
@@ -31,11 +31,10 @@
       <% end %>
     </ul>
   </div>
-
   <!-- Small layout end -->
 
   <!-- Large layout start -->
-  <table aria-describedby="table-description" class="hidden m:table crayons-table crayons-table--minimal w-100">
+  <table class="hidden m:table crayons-table crayons-table--minimal w-100" <%= @gdpr_delete_requests.empty? ? "aria-describedby='empty-state-message-id'".html_safe : "aria-describedby='table-description'".html_safe %>>
```

Comment:
Minor tweak, non-blocking:

I believe you could replace the following

```erb
<%= @gdpr_delete_requests.empty? ? "aria-describedby='empty-state-message-id'".html_safe : "aria-describedby='table-description'".html_safe
```

With this

```erb
aria-describedby="<%= @gdpr_delete_requests.empty? ? "empty-state-message-id" : "table-description" %>"
```

You need to call `html_safe` on the string because it has HTML attribute pair (key and value).  But with the change, you're not setting an attribute key and value but instead a value.

Reply:
Thank you both! I've updated this to reflect your suggestions. 😊